src/icons/Icons.stories.js

```diff
@@ -1,42 +1,76 @@
-import React from '@storybook/react'
-import { withKnobs, text, number, color } from '@storybook/addon-knobs'
+import React from 'react'
 
-const requireContext = require.context('.', true, /\.js?$/)
-const modules = requireContext.keys().filter((c) => !c.includes('.stories') && !c.includes('index.js'))
-const icons = modules.map((m) => ({
-  name: m.replace('./', '').replace('.js', ''),
-  Icon: requireContext(m).default
+import * as iconImports from './index.js'
+const icons = Object.keys(iconImports).map((m) => ({
+  name: m,
+  Icon: iconImports[m]
 }))
 
-const filterByTextQuery = (icon) => {
-  const searchQuery = text('Search', '')
-  return icon.name.includes(searchQuery)
+const filterByTextQuery = (icon, searchQuery) => {
+  return icon.name.toLowerCase().includes(searchQuery.toLowerCase())
+}
```

Comment:
had to change how we were loading icons because require.context isn't available. we export all icons from index.js, so importing them all from there.